pkgs/development/libraries/ffmpeg-full/default.nix

```diff
@@ -327,6 +330,8 @@ stdenv.mkDerivation rec {
     #(enableFeature avisynth "avisynth")
     (enableFeature (bzip2 != null) "bzlib")
     (enableFeature (celt != null) "libcelt")
+    (enableFeature cuda "cuda")
+    (enableFeature (clang != null && cuda-llvm) "cuda-llvm")
```

Comment:
Does `cuda-llvm` imply enabling `cuda`, or are they independent? Can they be enabled together without conflicts?

Reply:
Both of them should be enabled for full cuda features including nvdec hwaccel and cuda filters. `cuda-llvm` is the free alternative of the non-free `nvcc`.

Reply:
Both:
* `./configure --disable-cuda --enable-cuda-llvm`
* `./configure --enable-cuda --disable-cuda-llvm`

execute successfuly for me. But I am not sure whether it makes sense to have one, but not the other in a real-life scenario.

Reply:
I was just wondering that to see if there would be conflicts that would require assertions.

Reply:
> `cuda-llvm` is the free alternative of the non-free `nvcc`.

It doesn't look like `nvcc` is included here -- is it used dynamically when `cuda` is enabled if `cuda-llvm` is disabled?